db/migrate/20170627182248_drop_roles_if_exists.rb

```diff
@@ -0,0 +1,9 @@
+class DropRolesIfExists < ActiveRecord::Migration
+  def change
+    if table_exists?(:roles)
+      drop_table(:roles) do |t|
```

Comment:
I haven't used this syntax before. Is this to make it rollback-able? How does that interact with the `table_exists?` method?

This may be one where we want the rollback to be a no-op. It's not a big deal.